astropy/modeling/projections.py

```diff
@@ -216,8 +216,8 @@ class Pix2Sky_ZenithalPerspective(Pix2SkyProjection, Zenithal):
         Look angle γ in degrees.  Default is 0°.
     """
 
-    mu = Parameter(default=0.0)
-    gamma = Parameter(default=0.0, getter=_to_orig_unit, setter=_to_radian)
+    mu = Parameter(default=0.0, description="Distance from point of projection to center of sphere")
+    gamma = Parameter(default=0.0, getter=_to_orig_unit, setter=_to_radian, description="Look angle γ in degrees (Default = 0°)")
```

Comment:
Line break: (number of spaces is messed up in github comment, but you get the idea...
```
    gamma = Parameter(default=0.0, getter=_to_orig_unit, setter=_to_radian, 
              description="Look angle γ in degrees (Default = 0°)")
```

Reply:
See below., happens a few times that lines are very long. I think it's OK not to break the comment string, but you can certainly break between parameters.